serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -17,11 +19,36 @@ You can define a `dataInputSchema` as follows:
 }
 ----
 
-In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project.
+In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project or any other directory included in classpath.
+
+`failOnValidationErrors` is an optional flag that indicates the behaviour adopted when the input data does not match the specified JSON schema. If not specified or set to true, an exception will be thrown and flow execution fails. If set to false, the flow will be executed and a log of level WARN with the validation errors will be printed.
+
+== Output Schema
+
+Serverless Workflow specification does not support Json Output schema until version 0.9. Therefore {product_name} is implementing it as an link:{spec_doc_url}#extensions[Serverless Workflow specification extension]. Output schema is applied after workflow execution to verify that the output model has the expected format. It is also useful for Swagger generation purposes. 
+
+As with Input schema, you should specify the url to the json schema, using `outputSchema` as follows
```

Comment:
```suggestion
With Input schema, you must specify the URL to the JSON schema, using `outputSchema` as follows:
```

Reply:
@kaldesai here Im trying to same that the behaviour is similar to the input schema one, how should I write it?

Reply:
@fjtirado, maybe something like this?
```suggestion
Similar to Input schema, you must specify the URL to the JSON schema, using `outputSchema` as follows:
```